### ISSUE TITLE
question about _calculator property

### ISSUE THREAD

**Issue by rooberrydev** [jhart5/javascript-calculator#7]
As far as I can see this is the value to which  the dynamically calculated .calculator (which is a getter) writes?
Anything I'm missing?

That's really interesting, I hadn't seen this done before!

**Comment by JoshDAO** [comment]
Do you mean the _calculation property?

**Comment by JoshDAO** [comment]
So calculator._calculation is effectively the same as what is written in the main display box, but using '*' and '/' operators instead of '÷' and '×' which JavaScript understands and can evaluate. Once calculator._calculation is evaluated, the answer is displayed in calculator._calculatedDisplay.innerHTML. I should definitely create a setter function to update calculator._calculatedDisplay.innerHTML.

**Comment by rooberrydev** [comment]
oh yeah i meant _calculation! my brain probably typed calculator having read it so many times.

so these are the getter and setter for _calculation right?

```javascript
get calculation() {
        return this._calculation;
    },
    set calculation(value) {
        this._calculation += value;
    },
```

**Comment by JoshDAO** [comment]
Correct. So each time the user hits a button, the number/operator is concatenated to the end of the display (for the user to see) AND the _calculation property (for the JS to work on)
